### ISSUE TITLE
extra_installs() fails at the installation of refGenome

### ISSUE THREAD

**Issue by tillandlauer** [RajLabMSSM/echolocatoR#38]
Hi,

I've installed echolocatoR using conda and subsequently tried to install the additional dependencies using extra_installs(). This worked well for the non-archived packages but failed for refGenome in R 4.04 under macOS 11.2.3: 
```
In file included from ./refGenome.h:29:
./extptr.h:112:37: error: no matching function for call to 'Rf_unprotect'
        virtual ~atmptr() { if(protected_) UNPROTECT(pRob); }
/Library/Frameworks/R.framework/Resources/include/Rinternals.h:985:22: note: expanded from macro 'UNPROTECT'
#define UNPROTECT(n)    Rf_unprotect(n)
/Library/Frameworks/R.framework/Resources/include/Rinternals.h:1765:6: note: candidate function not viable: cannot convert
      argument of incomplete type 'SEXP' (aka 'SEXPREC *') to 'int' for 1st argument
void Rf_unprotect(int);
```
Is there a solution for getting the archived refGenome to compile and if not, which part of echolocatoR might not work correctly?

Thanks,
Till

**Comment by bschilder** [comment]
@tillandlauer , thanks for notifying me of this! it might be the case that these archived packages worked for older versions of R (<4.0) but are not compatible with the current version. I'll modify `extra_installs()` so it first tries to install XGR via Bioconductor ==> then CRAN ==> then archived, until it installs.

refGenome and foreign are both dependencies for XGR, so they should only affect XGR-related functions; which at this point is just the XGR arguments within `PLOT.locus()`.

**Comment by bschilder** [comment]
Now merged via #39
